### PR TITLE
Readmes alignment

### PR DESCRIPTION
**Pull request by Ayelet41**
Alignment between old Readme and User Guide website

### CODE REVIEW

File: website/content/en/docs/User Guide/Explore your project in GCP/_index.md [hunk 1]
```diff
@@ -30,9 +30,14 @@ In a new browser tab, navigate to the Hipster Shop URL, where you can "purchase"
...
 ## Run the load generator
+Cloud Ops Sandbox comes with [Locust load generator](https://locust.io/), to simulate users traffic.  
 
-In another browser tab, navigate to the load-generator URL, from which you can simulate users interacting with the application to generate traffic. For this application, values like 100 total users with a spawn rate of 2 users per second are reasonable. Fill in the **Host** field with the "Hipster shop web address" from the installation stage if it isn't prepopulated. Click the **Start swarming** button to begin generating traffic to the site.
+- In another browser tab, navigate to the load-generator URL(from the installation stage if it isn't populated).  
+- Enter the number of **users** and **spawn rate**. For this application, we recommend to test 100 total users with a spawn rate of 2 users per second.  
+- Fill in the **Host** field with the "Hipster shop web address" from the installation stage if it isn't populated.  
+- Click the **Start swarming** button to begin generating traffic to the site.
 
+This will produce traffic on the store from a loadgenerator pod:
```
Comment (simonz130) [inline website/content/en/docs/User Guide/Explore your project in GCP/_index.md:40]: Isn't it conflicting with your other PR where you make loadgen instruction changes?

Comment (Ayelet41) [reply website/content/en/docs/User Guide/Explore your project in GCP/_index.md]: It's not conflicting, but unneeded. Reverted.

File: website/content/en/docs/User Guide/Explore your project in GCP/_index.md [hunk 2]
```diff
@@ -30,9 +30,31 @@ In a new browser tab, navigate to the Hipster Shop URL, where you can "purchase"
...
+
+This will produce traffic on the store from a loadgenerator pod:
 
 ![Locust example](/docs/images/user-guide/3-locust.png)
 
-From here, you can explore how the application was deployed, and you can use the navigation menu to bring up other GCP tools.
\ No newline at end of file
+
+
+You can also run load testing from Cloud Shell using `sandboxctl` command, there are 2 available traffic patterns : 'basic' or 'step'
```
Comment (nicoleczhu) [inline website/content/en/docs/User Guide/Explore your project in GCP/_index.md:46]: Nit. add period at the end

File: website/content/en/docs/User Guide/Explore your project in GCP/_index.md [hunk 3]
```diff
@@ -30,9 +30,31 @@ In a new browser tab, navigate to the Hipster Shop URL, where you can "purchase"
...
+
+```bash
+sandboxctl loadgen <traffic_pattern>
+```
+
+A step load shape will increase user traffic in every step for the entire test duration([current settings](https://github.com/GoogleCloudPlatform/cloud-ops-sandbox/tree/master/src/loadgenerator/locust-tasks)).
```
Comment (nicoleczhu) [inline website/content/en/docs/User Guide/Explore your project in GCP/_index.md:52]: Nit: maybe rephrase to `For the entire test duration (see current settings), a load generator will generate step-shaped load profiles in the user traffic`? Is that close to what you're saying?

**Review by nicoleczhu — APPROVED** [review]
Left some nits , but LGTM overall! Thanks Ayelet!

File: website/content/en/docs/User Guide/Explore your project in GCP/_index.md (hunk 3 above)
Comment (Ayelet41) [reply website/content/en/docs/User Guide/Explore your project in GCP/_index.md:52]: Changed to 'For the entire `step test` duration ([current settings](https://github.com/GoogleCloudPlatform/cloud-ops-sandbox/tree/master/src/loadgenerator/locust-tasks)), a load generator will generate step-shaped load profiles in the user traffic.'
